acestep/training/lora_utils.py

```
@@ -251,48 +255,35 @@ def load_lora_weights(
-        lora_config: LoRA configuration (required if loading from .pt file)
+        lora_path: Path to saved LoRA adapter directory
+        lora_config: Unused; retained for API compatibility
         
     Returns:
         Model with LoRA weights loaded
     """
-    if not os.path.exists(lora_path):
-        raise FileNotFoundError(f"LoRA weights not found: {lora_path}")
+    safe_lora_path = _safe_checkpoint_dir(lora_path)
+    if safe_lora_path is None:
+        raise ValueError(f"Rejected unsafe LoRA path: {lora_path!r}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)